tuning/tuner_requirements/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pytest==8.2.2
+tqdm==4.66.4
+z3==0.2.0
```

Comment:
We don't need z3, it's some random unrelated package that happened to claim the name z3 first